webapp/install_mattermost_webapp.sh

```diff
@@ -0,0 +1,26 @@
+readonly COMMITHASH=8235b5bb44
```

Comment:
Any easy way to use whatever commit is in `../mattermost/webapp` if present? Just want to think about us poor MM developers :p

Reply:
I think that would probably cause issues with reproducibility -- I don't want my local calls build to depend on whatever I'm doing in my local server :)